ui/src/user/Login.tsx

```diff
@@ -44,7 +44,7 @@ class Login extends Component<Stores<'currentUser'>> {
                                 size="large"
                                 className="login"
                                 color="primary"
-                                disabled={!this.props.currentUser.hasNetwork}
+                                disabled={this.props.currentUser.connectionErrorMessage}
```

Comment:
disable expects a boolean, you can double negate the string to get the boolean value
```suggestion
                                disabled={!!this.props.currentUser.connectionErrorMessage}
```
(this is the reason why the build fails)